cmd/tink-worker/client/self-signed-cert.pem

```diff
@@ -0,0 +1,12 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Might want to move this to a directory called `testdata`. The Go tools ignore any directories with this name. [ref](https://pkg.go.dev/cmd/go/internal/test)

Reply:
This belongs in a constant in the unit test files.

Reply:
I've taken the approach Stephen suggested here and saved it as a constant instead.